src/Common/src/Common.Http/Discovery/DiscoveryHttpClientHandler.cs

```diff
@@ -8,6 +8,7 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Steeltoe.Common.LoadBalancer;
```

Comment:
Silly minor, but this needs to be sorted alphabetically (move line 11 up to line 6)